recipes/full_finetune_distributed.py

```diff
@@ -202,6 +203,7 @@ def setup(self, cfg: DictConfig) -> None:
 
         checkpoint_dict = self.load_checkpoint(cfg_checkpointer=cfg.checkpointer)
 
+        self._model_compile = cfg.get("compile", False)
```

Comment:
We should definitely land distributed FFT model compile as a fast follow here; until then the behavior will be confusing to folks